src/backend/tests/it/config.rs

```diff
@@ -25,6 +26,18 @@ fn config_is_available_to_allowed_users_only() {
         update_call::<UserProfile>(&pic_setup, nns_root, "config", ()).is_err(),
         "NNS root should not be able to call config"
     );
+    // Try a controller
+    assert_eq!(
+        update_call::<Config>(
+            &pic_setup,
+            Principal::from_text(CONTROLLER)
```

Comment:
You don't want to reuse the expose `controller()` function of the pocket ic utils here? or on the contrary this particular function should be not public?

Reply:
Good catch.  Done.